Introspective.Web/Views/NikoNikoEntry/List.cshtml

```diff
@@ -1,61 +1,77 @@
 ﻿@model IEnumerable<Introspective.ViewModels.NikoNikoEntryViewModel>
 
 @{
-    ViewBag.Title = "List NikoNikoEntrys";
+    ViewBag.Title = "List Niko Niko Entrys";
```

Comment:
Should this be "Niko Niko Entries"?

There's a few instances of this spelling through these pages

Reply:
Perfect, changed it. Thanks!